cou/steps/backup.py

```diff
@@ -43,7 +44,7 @@ async def backup(model_name: Optional[str] = None) -> str:
     logger.info("Set permissions to read mysql-innodb-cluster:%s ...", basedir)
     await utils.async_run_on_unit(unit_name, f"chmod o+rx {basedir}", model_name=model_name)
 
-    local_file = os.path.abspath(os.path.basename(remote_file))
+    local_file = str(pathlib.Path(os.getenv("COU_DATA", ""), os.path.basename(remote_file)))
```

Comment:
not-blocking: can we use only pathlib here? And I believe that `local_file` could be type `Path` instead of `str`.

Reply:
done